<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>413</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>835</width>
    <height>427</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JATT- Java Auto Tuning Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>490</width>
      <height>361</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>490</width>
      <height>290</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>481</width>
       <height>331</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>270</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_tuning_command">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tuning Command</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_runtime_fraction">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Wait time fraction to default time</string>
      </property>
      <property name="text">
       <string>Runtime fraction</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_tuningcommand">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>311</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_runtime_fraction">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>311</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>461</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="cmd_stop">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>20</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>   Stop</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/off.svg</normaloff>../icons/off.svg</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="cmd_test">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>   Start</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/start.jpeg</normaloff>../icons/start.jpeg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>-20</y>
        <width>20</width>
        <height>281</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="cmd_browse_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>120</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/open.jpeg</normaloff>../icons/open.jpeg</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_congifuration_file">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Configuration file path</string>
      </property>
      <property name="text">
       <string>Configuration file</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_configfile">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>281</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_source">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Source </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_source">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>281</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Target program to tune</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cmd_browse_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>70</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/open.jpeg</normaloff>../icons/open.jpeg</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>211</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>JVM-Tuner Configuration</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>321</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>281</width>
       <height>331</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>145</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="chk_bytecode">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Byte Code</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_codecache">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Code Cache</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_jit">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>JIT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_memory">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Memory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_deoptimization">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Deoptimization</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>121</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="chk_interpreter">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Interpreter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_properties">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Properties</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_temporary">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Temporary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_gc">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>GC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_all">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>All</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>20</width>
       <height>351</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>211</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string> JVM Flags</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
